setjmp/libpng/config.json

```diff
@@ -0,0 +1,38 @@
+{
+    "ana" : {
+        "base" : {
+            "context" : {
+                "non-ptr" : false
+            }
+        },
+        "malloc" : {
+            "wrappers" : ["png_malloc","png_malloc_warn","png_malloc_default",
+                "png_malloc_base","png_malloc_array","png_malloc_array_checked",
+                "png_calloc","png_zalloc","png_create_png_struct"],
+            "unique_address_count" : 1
+        },
+        "dead-code" :{
+            "branches" : false
+        }
+    },
+    "sem": {
+        "unknown_function" : {
+            "spawn" : false,
+            "invalidate" : {
+                "globals" : false,
+                "args" : false
+            }
+        }
+    },
+    "dbg" : {
+        "verbose" : true
+    },
+    "exp" : {
+        "volatiles_are_top" : false
+    },
+    "warn" : {
+        "info" : false,
+        "imprecise" : false,
```

Comment:
This will have to be removed, as the warning for completely unknown jumpbuffers is in this category.